ts/src/header-validator/validate-json.ts

```diff
@@ -17,13 +17,24 @@ const hex128Regex = /^0[xX][0-9A-Fa-f]{1,32}$/
 
 const UINT32_MAX: number = 2 ** 32 - 1
 
+export type ParseOpts = {
+  parseEventLevelEpsilon: boolean
```

Comment:
Can you help me understand why you added this as an optional thing?

Reply:
It corresponds to whether the Chromium implementation will disable the feature by default. If we enable instead, I will remove this option.

Reply:
ack